python/lsst/ts/standardscripts/maintel/integration_tests/ccw_rotator_stress_test.py

```diff
@@ -0,0 +1,266 @@
+
+        random_pos = np.random.random(self.n_rot_pos) * 80.0
+
+        self.log.debug(
+            f"Starting first test. Selected random rotator angles: {random_pos}"
+        )
+
+        for rotator_angle in random_pos:
+
+            await self.checkpoint(f"rotator_angle: {rotator_angle}")
+
+            self.log.debug(f"Testing rotator angle {rotator_angle}")
+
+            for i in range(self.n_pos_repeat):
+
+                await self.checkpoint(f"cycle {i+1} of {self.n_pos_repeat}")
+
+                self.log.debug("Slewing to + position.")
+
+                time_and_date = await self.mtcs.rem.mtptg.tel_timeAndDate.aget()
+                ra = Angle(time_and_date.lst, unit=u.hourangle)
+                self.mtcs.rem.rotator.evt_inPosition.flush()
+                await self.mtcs.slew_icrs(
+                    ra=ra,
+                    dec=dec,
+                    rot_sky=-180.0 - rotator_angle,
+                    stop_before_slew=False,
+                )
+
+                self.log.debug(
+                    f"Arrived at + position, tracking for {self.track_time}s"
+                )
+                await asyncio.sleep(self.track_time)
+
+                self.log.debug("Slewing to - position.")
+
+                time_and_date = await self.mtcs.rem.mtptg.tel_timeAndDate.aget()
+                ra = Angle(time_and_date.lst, unit=u.hourangle)
+                self.mtcs.rem.rotator.evt_inPosition.flush()
+                await self.mtcs.slew_icrs(
+                    ra=ra,
+                    dec=dec,
+                    rot_sky=-180.0 + rotator_angle,
+                    stop_before_slew=False,
+                )
+
+                self.log.debug(
+                    f"Arrived at - position, tracking for {self.track_time}s"
+                )
+                await asyncio.sleep(15.0)
+
+        await self.checkpoint("reset_rotator")
+        self.log.info("First test completed. Reseting rotator.")
+
+        time_and_date = await self.mtcs.rem.mtptg.tel_timeAndDate.aget()
+        ra = Angle(time_and_date.lst, unit=u.hourangle)
+        await self.mtcs.slew_icrs(
+            ra=ra, dec=dec, rot_sky=-180.0, stop_before_slew=False
+        )
+
+        self.log.debug(f"Arrived at - position, tracking for {self.track_time}s")
+        await asyncio.sleep(self.track_time)
+
+        self.log.debug(f"Starting second test. Time delays: {self.time_delays}")
+
+        for delay in self.time_delays:
+
+            await self.checkpoint(f"delay {delay}s.")
+            self.log.debug(f"Test delay: {delay}s.")
+
+            time_and_date = await self.mtcs.rem.mtptg.tel_timeAndDate.aget()
+            ra = Angle(time_and_date.lst, unit=u.hourangle)
+            try:
+                await self.mtcs.slew_icrs(
+                    ra=ra,
+                    dec=dec,
+                    rot_sky=-180.0 + 80.0,
+                    stop_before_slew=False,
+                    slew_timeout=delay,
+                )
```

Comment:
This will stop working in ts_salobj v6, because the slew command will extend the timeout based on the reported INPROGRESS acknowledgement.

Here is one simple fix that will work with salobj 5 and 6 (don't specify a value for the `slew_timeout` argument):
```
await asyncio.wait_for(self.mtcs.slew_icrs(...), timeout=delay)
```